resources/icons/16px/globe-network-add.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M6.69254 5C6.80936 5.28504 6.98141 5.55273 7.21434 5.78566C7.29258 5.8639 7.37475 5.93528 7.46018 6H6.12C6.05 6.61 6 7.27 6 8C6 8.73 6.05 9.39 6.12 10H9.88C9.93272 9.54062 9.97409 9.0529 9.99131 8.52827C10.0583 8.61785 10.1326 8.70389 10.2143 8.78566C10.435 9.00637 10.687 9.17242 10.9553 9.28861C10.938 9.532 10.9161 9.76878 10.89 10H13.65C13.8087 9.54551 13.9206 9.0702 13.9706 8.57782C14.3352 8.11636 14.5 7.55453 14.5 7V6.5H15C15.2807 6.5 15.5633 6.45776 15.8337 6.36877C15.9427 6.89538 16 7.44097 16 8C16 12.42 12.42 16 8 16C3.58 16 0 12.42 0 8C0 3.58 3.58 0 8 0C8.55903 0 9.10462 0.0572697 9.63123 0.166261C9.54224 0.436735 9.5 0.719302 9.5 1V1.5H9C7.47578 1.5 6.5 2.77073 6.5 4C6.5 4.01614 6.50014 4.0323 6.50042 4.04845C6.41447 4.33822 6.33699 4.656 6.27 5H6.69254ZM5.27 5C5.47 3.89 5.78 2.97 6.18 2.31C4.75 2.77 3.57 3.74 2.83 5H5.27ZM2.35 6C2.13 6.63 2 7.3 2 8C2 8.7 2.13 9.37 2.35 10H5.11C5.04 9.38 5 8.72 5 8C5 7.28 5.04 6.62 5.11 6H2.35ZM5.27 11H2.83C3.57 12.26 4.75 13.23 6.18 13.69C5.78 13.03 5.48 12.11 5.27 11ZM6.27 11C6.64 12.9 7.33 14 8 14C8.67 14 9.36 12.9 9.73 11H6.27ZM10.73 11C10.52 12.11 10.22 13.03 9.82 13.69C11.25 13.23 12.43 12.26 13.17 11H10.73Z" fill="#111418"/>
+<path d="M9 5C8.4 5 8 4.6 8 4C8 3.5 8.4 3 9 3H11V1C11 0.4 11.4 0 12 0C12.6 0 13 0.4 13 1V3H15C15.6 3 16 3.4 16 4C16 4.6 15.6 5 15 5H13V7C13 7.6 12.6 8 12 8C11.4 8 11 7.6 11 7V5H9Z" fill="#111418"/>
+</svg>
```

Comment:
Can we clean this up a bit? We can safely remove the width/height dimensions, fill/clip-rules, and fills:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
    <path d="M6.693 5a2.4 2.4 0 0 0 .767 1H6.12C6.05 6.61 6 7.27 6 8s.05 1.39.12 2h3.76c.053-.46.094-.947.111-1.472a2.4 2.4 0 0 0 .964.76q-.026.365-.065.712h2.76c.159-.454.27-.93.32-1.422A2.53 2.53 0 0 0 14.5 7v-.5h.5c.28 0 .563-.042.834-.131q.165.79.166 1.631c0 4.42-3.58 8-8 8s-8-3.58-8-8A7.998 7.998 0 0 1 9.631.166 2.7 2.7 0 0 0 9.5 1v.5H9C7.476 1.5 6.5 2.77 6.5 4v.048q-.13.435-.23.952zM5.27 5c.2-1.11.51-2.03.91-2.69A6.03 6.03 0 0 0 2.83 5zM2.35 6C2.13 6.63 2 7.3 2 8s.13 1.37.35 2h2.76C5.04 9.38 5 8.72 5 8s.04-1.38.11-2zm2.92 5H2.83a6.03 6.03 0 0 0 3.35 2.69c-.4-.66-.7-1.58-.91-2.69m1 0c.37 1.9 1.06 3 1.73 3s1.36-1.1 1.73-3zm4.46 0c-.21 1.11-.51 2.03-.91 2.69A6.03 6.03 0 0 0 13.17 11z"/>
    <path d="M9 5c-.6 0-1-.4-1-1 0-.5.4-1 1-1h2V1c0-.6.4-1 1-1s1 .4 1 1v2h2c.6 0 1 .4 1 1s-.4 1-1 1h-2v2c0 .6-.4 1-1 1s-1-.4-1-1V5z"/>
</svg>
```

Reply:
Fixed! Thanks